Makefile

```diff
@@ -1,52 +1,191 @@
-DEFAULT_GOARCH := $(shell go env GOARCH)
-BUILD_HASH = $(shell git rev-parse HEAD)
-LDFLAGS += -X "github.com/mattermost/rtcd/service.buildHash=$(BUILD_HASH)"
-
-## Check go mod files consistency
-.PHONY: gomod-check
-gomod-check:
-	@echo Checking go mod files consistency
-	go mod tidy -v && git --no-pager diff --exit-code go.mod go.sum || (echo "Please run \"go mod tidy\" and commit the changes in go.mod and go.sum." && exit 1)
-
-.PHONY: check-style
-check-style: golangci-lint gomod-check
-	@echo Checking for style guide compliance
-
-.PHONY: golangci-lint
-golangci-lint: ## Run golangci-lint on codebase
-# https://stackoverflow.com/a/677212/1027058 (check if a command exists or not)
-	@if ! [ -x "$$(command -v golangci-lint)" ]; then \
-		echo "golangci-lint is not installed. Please see https://github.com/golangci/golangci-lint#install for installation instructions."; \
-		exit 1; \
-	fi; \
-
-	@echo Running golangci-lint
-	golangci-lint run ./...
 
-.PHONY: test
-test:
-	go test -v -mod=readonly -failfast -race ./...
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE for license information.
+
+# ====================================================================================
+# Variables
+# App Go Code location
+CONFIG_APP_CODE        += ./cmd/rtcd
+# Dockerfile's location
+CONFIG_DOCKERFILE      += ./build/Dockerfile
+# Docker options to inherit for all docker run commands
+CONFIG_DOCKEROPTS      += --rm -i -u $$(id -u):$$(id -g) --platform "linux/amd64"
+# Registry to upload images
+CONFIG_IMAGE_REGISTRY  += my.registy.com
```

Comment:
I think it will be better to have here by default the docker hub

Reply:
Rebased and pushed changes